### ISSUE TITLE
unable to train VGG16

### ISSUE THREAD

**Issue by efferre99** [keras-team/keras-applications#40]
I'm trying to build a VGG16 model (without using the pre-trained one), but 25 epochs aren't enough because I get overfitting. How many epochs are enough? I have to classify two object
sgd = optimizers.SGD(lr=0.0001, decay=1e-6, momentum=0.9)
model.compile(loss='categorical_crossentropy', # Better loss function for neural networks
optimizer=sgd, # Adam optimizer with 1.0e-4 learning rate
metrics = ['accuracy']) # Metrics to be evaluated by the model
model_details=model.fit_generator(train_batches,steps_per_epoch=3,validation_data=valid_batches,validation_steps=3, epochs=25,verbose=2)

**Comment by taehoonlee** [comment]
@efferre99, The convergence rate depends on the data. You can refer to the following cases ImageNet:
- https://github.com/ruimashita/caffe-train/blob/master/vgg.solver.prototxt (max_iter: 450000)
- https://github.com/pherrusa7/Caffe-Utilities/blob/master/VGG/solver.prototxt (max_iter: 600000)

**Comment by taehoonlee** [comment]
@efferre99, I'll close the issue for now. Please feel free to open it again at any time if you have additional comments.
